Makefile

```diff
@@ -142,6 +142,15 @@ EXTRA_LD_FLAGS  :=
 #
 # Default Tool options - can be overridden in {mcu}.mk files.
 #
+DEBUG_MIXED = no
```

Comment:
DEBUG_MIXED ?= no

Reply:
IMO not necessary - Option from command line overrides Makefile assignment (https://www.gnu.org/software/make/manual/html_node/Overriding.html). So ?= is necessary only if setting by environment variable is necessary.